include/picongpu/plugins/output/param.hpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright 2013-2023 Rene Widera
+/* Copyright 2013-2023 Axel Huebl, Rene Widera, Sergei Bastrakov, Klaus Steiniger
```

Comment:
authors added

Reply:
github is mixing removed and newly created files.